src/cr/cube/matrix/measure.py

```diff
@@ -318,6 +323,27 @@ def _weighted_cube_counts(self):
         return self._cube_measures.weighted_cube_counts
 
 
+class _CellPopulationFractions(_BaseSecondOrderMeasure):
+    """Provides the cell-specific fraction of population
+
+    If any of the dimensions (rows or columns) is a categorical date, the
+    appropriate percentages are used, to calculate the population counts, so as to
+    be the total amount among categorical dates.
+
+    Otherwise, table percents are used to calculate population counts.
+    """
+
+    @lazyproperty
+    def blocks(self):
+        return (
+            self._second_order_measures.row_proportions.blocks
+            if self._dimensions[0].dimension_type in (DT.CAT_DATE, DT.CA_SUBVAR)
+            else self._second_order_measures.column_proportions.blocks
+            if self._dimensions[1].dimension_type in (DT.CAT_DATE, DT.CA_SUBVAR)
+            else self._second_order_measures.table_proportions.blocks
+        )
```

Comment:
You'll need to choose between putting this logic here or up in assembler. I think the choice comes down to whether we need it for sort-by-value, which I expect Mike will want. In that case, the assembler bit would just become `self._assemble_matrix(self._measures.population_proportions.blocks)` and this would stay.

You would of course need an entry in `SecondOrderMeasures` for it then.

Docstring.